### PR TITLE
Revive our CI

### PR DESCRIPTION
**Pull request by ari-aviran**

### CODE REVIEW

File: .github/workflows/csp-temp-ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Todo !! delete before merge to elastic/kibana!!
+name: CSP temp Pre Merge CI
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (orouz) [inline .github/workflows/csp-temp-ci.yml:7]: if `main` is still our merge target then this is ok (maybe not needed, depends if it was included/excluded in the PR)

if it isn't out merge target, than we might need to add that branch name here?

File: .github/workflows/csp-temp-ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - name: kbn bootstrap
+        run: yarn kbn bootstrap
+
+      - name: TS
+        run: node_modules/.bin/tsc --build x-pack/plugins/cloud_security_posture/tsconfig.json --pretty
+
+      # - name: check TS
+      #   run: node scripts/type_check
+
+      - name: eslint
```
Comment (orouz) [inline .github/workflows/csp-temp-ci.yml:33]: maybe lets add `eslint_with_types` as we failed on that 

perhaps other CI checks that's easy to add

**Review by orouz — APPROVED** [review]

File: .github/workflows/csp-temp-ci.yml (hunk 1 above)
Comment (ari-aviran) [reply .github/workflows/csp-temp-ci.yml:7]: it is

File: .github/workflows/csp-temp-ci.yml (hunk 2 above)
Comment (ari-aviran) [reply .github/workflows/csp-temp-ci.yml:33]: i dont want to invest time in this at the moment

**Pull request merged**